components/token/TokenSelectDropdown.vue

```diff
@@ -38,7 +38,7 @@
             </CommonCardWithLineButtons>
           </div>
         </template>
-        <CommonEmptyBlock v-else class="search-empty-block">
+        <CommonEmptyBlock v-else class="search-empty-block" data-testId="warning_modal">
```

Comment:
@obmatter Generally we are awaiting for an approve from Daniil side if smth changes in vue components.

Reply:
Thank you - Danil is currently OOO - so do we need to wait for him?

Reply:
@pcheremu resolving this thread - as we don't need this testId - based on the fact that we changed the approach to the test